### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by matsn0w** [matsn0w/IIS-EmergencyLS-ELS-FiveM#164] [Announcements]
## Updating

We moved from NativeUI to [WarMenu](https://github.com/warxander/warmenu). You don't have to remove NativeUI (you might need it for some other resource), but make sure you have WarMenu installed!

Also see the [general Updating guide](/docs/installation.md#upgrading).

## What's Changed
* feature: flash high beam by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/89
* feature: add stop sign beeping by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/101
* Create FUNDING.yml by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/106
* feature: switch to personal keybinds by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/109
* docs: add more info about the 'move-over' question in the FAQ by @ImperiousTT in https://github.com/matsn0w/MISS-ELS/pull/112
* fix: disable auto repair when toggling extras by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/114
* feat: only disable siren when all lights are turned off by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/115
* fix: remove accidental object key by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/116
* feat: add high beams toggle to menu, refactor code by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/118
* feat: move from NativeUI to WarMenu by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/120
* feat: support miscs as light sources by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/152
* v2.2.0 by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/90

## New Contributors
* @ImperiousTT made their first contribution in https://github.com/matsn0w/MISS-ELS/pull/112

**Full Changelog**: https://github.com/matsn0w/MISS-ELS/compare/2.1.1...2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/matsn0w/MISS-ELS/releases/tag/2.2.0'>v2.2.0</a>.</em>

**Comment by dutchxarms** [comment]
when people use sirens and sound people lagg af.
